portal/v2/src/ClusterDetailListComponents/ClusterOperatorList.tsx

```diff
@@ -0,0 +1,188 @@
+import * as React from 'react';
+import { useState, useEffect } from "react"
+import { Stack, StackItem, IconButton, IIconStyles, SelectionMode } from '@fluentui/react';
+import { Link } from '@fluentui/react/lib/Link';
+import { IColumn } from '@fluentui/react/lib/DetailsList';
+import { ShimmeredDetailsList } from '@fluentui/react/lib/ShimmeredDetailsList';
+import { IClusterOperator } from './ClusterOperatorsWrapper';
+import { ClusterOperatorsComponent } from './ClusterOperator';
+
+export declare interface IClusterOperatorList {
+  name: string;
+  available: string;
+}
+
+interface ClusterOperatorComponentProps {
+  clusterOperators: any
```

Comment:
Another place it would be better to specify a concrete type or `unknown` if possible.